apis/proto/payload/payload.proto

```diff
@@ -141,7 +172,6 @@ message Backup {
 
   message MetaVector {
```

Comment:
come to think of it, because the `meta` field is deleted in this PR, the naming of this message becomes different from as it is :joy:
how do you think about it?

Reply:
fixed